packages/TorneloScoresheet/__tests__/useAppModeState.test.ts

```diff
@@ -119,7 +119,7 @@ describe('graphical recording moving', () => {
 
     const graphicalState = generateGraphicalRecordingState([]);
```

Comment:
This test utility function needs to be renamed, and the variables should probably be changes to recordingState everywhere (you can easily do a find and replace) otherwise it might be confusing for people maintaining this code later on...